frontend/app/reservations/_source/components/stage/VenueMap.tsx

```
@@ -0,0 +1,72 @@
+
+  return (
+    <div className="relative w-127.5 h-108.75">
+      <>
+        <style>{dynamicStyles}</style>
+        <div
+          className="w-full h-full [&>svg]:w-full [&>svg]:h-full select-none pointer-events-auto"
+          dangerouslySetInnerHTML={{
+            __html: svgContent ? svgContent : "",
+          }}
+          onClick={handleStageClick}
+        />
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)